packages/core/data-transfer/src/file/providers/source/index.ts

```diff
@@ -143,13 +149,16 @@ class LocalFileSourceProvider implements ISourceProvider {
               return false;
             }
             return isFilePathInDirname('assets/uploads', filePath);
+            // isFilePathInDirname('assets/metadata', filePath)
           },
-          onentry(entry) {
+          async onentry(entry) {
             const { path: filePath, size = 0 } = entry;
             const normalizedPath = unknownPathToPosix(filePath);
             const file = path.basename(normalizedPath);
+            const metadata = await loadAssetMetadata(`assets/metadata/${file}.json`);
```

Comment:
Can we try/catch this and still support backups that don't include the metadata folder? And in those cases we would know it's all local media.